shared/settings/index.stories.tsx

```diff
@@ -9,10 +9,12 @@ import password from './password/index.stories'
 import chat from './chat/index.stories'
 import landing from './landing/index.stories'
 import files from './files/index.stories'
+import contacts from './contacts-joined/index.stories.native'
```

Comment:
I would not expect this to work forever. Can you add a `.desktop` version of this that noops?